torch/testing/_internal/common_distributed.py

```diff
@@ -68,17 +68,14 @@ def wrapper(*args, **kwargs):
 
 def skip_if_not_multigpu(func):
```

Comment:
looks good. I noticed the problem is that the func argument in def skip_if_not_multigpu. If you remove the argument it works as intended. also matches the pattern of the other defs with def decorator(func): @wraps(func)

Reply:
The problem is that `skip_if_not_multigpu` used the double wrapper structure which is meant for parametrizable decorators. Removing one wrapper would have been sufficient. See my proposal in #53502.